back/api/models.py

```diff
@@ -202,6 +202,7 @@ class OrderStatus(models.TextChoices):
 
     class Meta:
         db_table = 'order'
+        ordering = ['-date_ordered']
```

Comment:
Does that mean that data is ordered in descending order using attribute `date_ordered` ?